packages/wdio-browserstack-service/src/Percy/PercySDK.ts

```diff
@@ -0,0 +1,46 @@
+import { createRequire } from 'node:module'
+const require = createRequire(import.meta.url)
```

Comment:
Let's not use require and instead do `await import(packageName)`

Reply:
Done